command/ca/provisioner/add.go

```diff
@@ -74,104 +137,167 @@ func addAction(ctx *cli.Context) (err error) {
 		return errs.RequiredFlag(ctx, "ca-config")
 	}
 
-	var password string
-	if passwordFile := ctx.String("password-file"); len(passwordFile) > 0 {
-		password, err = utils.ReadStringPasswordFromFile(passwordFile)
-		if err != nil {
-			return err
-		}
-	}
-
 	c, err := authority.LoadConfiguration(config)
 	if err != nil {
 		return errors.Wrapf(err, "error loading configuration")
 	}
 
-	var provisioners []*authority.Provisioner
-	provMap := make(map[string]*authority.Provisioner)
-	for _, prov := range c.AuthorityConfig.Provisioners {
-		provisioners = append(provisioners, prov)
-		provMap[prov.Name+":"+prov.Key.KeyID] = prov
+	typ := strings.ToUpper(ctx.String("type"))
```

Comment:
in the docs you say this is case-sensitive, but it doesn't appear to be.

Reply:
It says `Type is a case-insensitive string`

Reply:
I'm bad at reading.